eng/pipelines/test-integration-job.yml

```diff
@@ -29,7 +29,7 @@ steps:
 
   - script: eng\cibuild.cmd
       -configuration ${{ parameters.configuration }}
-      -msbuildEngine dotnet
+      -msbuildEngine vs
```

Comment:
This is the fix for the failure to deploy. [Arcade does not import the targets](https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/Imports.targets#L22) required to deploy the VSIX when using the dotnet msbuildEngine.